src/components/app/app.html

```diff
@@ -1 +1,8 @@
-<h1>Hi!</h1>
\ No newline at end of file
+<div class="app">
+  <transaction-dialog show="$ctrl.transactionDialogShow"></transaction-dialog>
+  <md-button class="app__add-transaction-dialog-activation md-fab md-primary" aria-label="FAB" ng-click="$ctrl.onAddTransactionClick($event)">
```

Comment:
please do not use such long class names; BTW I thought that this is just `transaction-dialog` without `add`